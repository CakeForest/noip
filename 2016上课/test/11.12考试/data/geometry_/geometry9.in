100
230 776 726 456 260 268 449 650 76 104 140 132 530 541 262 740 128 184 954 136 420 600 23 378 315 146 906 200 880 712 805 208 992 138 696 624 784 16 463 715 940 16 340 537 613 370 12 682 906 884 493 28 112 952 848 476 120 510 672 360 125 302 70 231 37 295 104 404 447 600 814 761 891 496 325 418 68 480 161 528 148 420 791 104 661 829 505 815 380 528 314 305 305 778 128 322 737 714 713 20 
200 756 494 245 204 605 897 400 290 380 719 314 948 100 748 262 545 0 80 683 936 160 58 520 520 658 340 760 301 576 396 559 740 964 176 689 860 480 500 550 200 388 179 581 71 712 344 591 48 236 712 400 632 600 384 948 586 594 560 96 400 970 940 978 687 844 466 746 838 616 391 120 240 998 18 926 425 872 500 432 132 136 700 880 544 490 158 774 712 750 870 222 372 883 94 160 565 200 439 557 
100
181 756
942 200
213 450
470 600
520 810
587 956
576 897
862 228
996 953
640 154
774 629
300 578
360 860
946 616
152 800
420 120
32 747
747 908
364 573
970 46
936 312
25 916
152 876
685 608
842 520
874 980
160 747
202 448
592 760
290 992
844 334
410 644
524 440
660 561
987 271
161 940
0 948
606 936
100 568
968 966
460 175
259 266
418 896
680 984
218 116
841 760
412 210
502 910
618 689
257 970
110 886
184 837
800 736
855 911
848 892
640 96
874 816
618 76
490 125
552 696
912 655
493 325
11 852
130 137
235 160
230 808
423 72
912 235
482 896
586 510
511 77
0 636
534 680
524 982
632 953
284 185
380 936
432 44
552 701
560 244
848 53
144 104
261 787
16 152
385 939
744 9
417 400
458 480
444 16
627 652
828 128
387 534
532 208
73 636
230 0
990 50
650 572
673 64
70 957
13 130
