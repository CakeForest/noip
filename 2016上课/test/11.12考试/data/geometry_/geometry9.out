94
100
69
100
100
100
100
100
100
84
100
90
100
100
96
61
83
100
96
100
100
94
100
100
100
100
91
68
100
100
100
100
98
100
100
100
96
100
68
100
68
57
100
100
37
100
67
100
100
100
99
100
100
100
100
79
100
73
67
100
100
86
86
29
43
100
56
100
100
100
65
67
100
100
100
50
100
54
100
85
94
28
100
15
100
81
85
96
53
100
98
94
79
73
29
100
100
79
100
12
